slices/libcbor0.10.yaml

```diff
@@ -0,0 +1,15 @@
+package: libcbor0.10
+
+essential:
+  - libcbor0.10_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/lib/*-linux-*/libcbor.so.0.10*:
```

Comment:
well done. you didn't fall into the temptation of doing `libcbor.so.0*` :stuck_out_tongue: